scripts/vet.sh

```diff
@@ -119,7 +119,6 @@ grep "(SA1019)" "${SC_OUT}" | not grep -Fv 'XXXXX PleaseIgnoreUnused
 XXXXX Protobuf related deprecation errors:
 "github.com/golang/protobuf
 .pb.go:
-grpc_testing_not_regenerate
```

Comment:
Staticcheck is staying within the module?

Ugh, should we be running it for every `go.mod` then?

Reply:
That's right. Let's track this here https://github.com/grpc/grpc-go/issues/7141